AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/dtos/DeploymentDTO.java

```diff
@@ -68,28 +69,72 @@ public DeploymentDTO(DeploymentEntity entity) {
 		this.trackingId = entity.getTrackingId();
 		this.state = entity.getDeploymentState();
 		this.appServerName = entity.getResourceGroup().getName();
-		this.appServerId = entity.getResource().getId();
+		this.appServerId = entity.getResourceGroup().getId();
```

Comment:
You can't change this, this is will change the rest api.
Please make a new field appServerResourceId and assign it entity.getResourceGroup().getId()
This also means you have to add a new field in the TypeScript entities.

Reply:
@yvespp appServerId has been resourceGroup.id until yesterday evening - so I basically reverted my last change which led to the redeployment issue

Reply:
Ok, in 1.14 it is the groupId and has been since it was introduced in 1.12 (for the create deployment angular screen?).
Still, the name appServerId is unfortunate and we should think about deprecating it to replace it with a new field.